<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesOrder</class>
 <widget class="QWidget" name="SalesOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_SalesOrder_id">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер Заказа продаж</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SalesOrder_id">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_customer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Покупатель (реквизиты)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_customer">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>421</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Контактное лицо</string>
   </property>
   <widget class="QLabel" name="label_contactName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Фамилия Имя Отчество</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_contactName">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_post">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Должность</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_post">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_phone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Телефон</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phone">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Электронная почта</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_dateReg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата регистрации</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_reg">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата отгрузки</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_shipment">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>350</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>421</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Договор</string>
   </property>
   <widget class="QLabel" name="label_contractNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Номер договора</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_contractNumber">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_dateContract">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата договора</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_dateContract">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_approval">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>530</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Согласие Отдела продаж</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_yes">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>530</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Да</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_no">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>530</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Нет</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_managerName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Менеджер продаж</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_managerName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>580</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>431</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Итого</string>
   </property>
   <widget class="QLabel" name="label_totalWithoutDiscount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Итого без скидки на ГП на продажу (руб.)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_discount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Скидка на ГП на продажу (руб.)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>НДС</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Итого</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_totalWithoutDiscount">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_discount">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vat">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_total">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>790</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>790</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
